CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # ChangeLog
 
+## 0.5.0 (unreleased)
+
+- bump minimum Rust version to 1.32.0
```

Comment:
Gotta stick with 1.31.0 to stay compatible with https://crates.io/crate/log

Reply:
Yup, happy to stick with 1.31 to match `log`'s MSRV. We'll need to fix our docs to mention the requirement for `log >= 0.4.10` instead of `0.4.1`.

Reply:
Latest commit should have updated all the docs.